src/utils/updaters/updateEventElementStyle.ts

```diff
@@ -0,0 +1,17 @@
+import { CalendarInfo, ExtendedEvent } from "../../types.js";
+import { getTableCell } from "../getters/getTableCell.js";
+import { getEventTopPos } from "../getters/getEventTopPos.js";
+import { getEventLeftPos } from "../getters/getEventLeftPos.js";
+import { getEventHeight } from "../getters/getEventHeight.js";
+import { getEventWidth } from "../getters/getEventWidth.js";
+import { getEventZIndex } from "../getters/getEventZIndex.js";
+
+export function updateEventElementStyle(eventElem: HTMLDivElement, extEvent: ExtendedEvent, calendarInfo: CalendarInfo): void {
```

Comment:
I noticed where you use this function, you query for the `eventElem` element before calling it. Why not just do that here and not pass an element in the parameters ?
The function would become more independent and reusable.
So instead of:
```
const filteredExtEvents = getFilteredExtEvents(startDate, calendarInfo);
	filteredExtEvents.forEach(event => {
		const eventElem = document.querySelector(`[data-event-id='${event.eventId}']`) as HTMLDivElement;
		updateEventElementStyle(eventElem, event, calendarInfo);
	});
```

You could:
```
const filteredExtEvents = getFilteredExtEvents(startDate, calendarInfo);
filteredExtEvents.forEach(event => updateEventElementStyle(event, calendarInfo));
```